### ISSUE TITLE
Symbol Token should not be the Primary Key

### ISSUE THREAD

**Issue by manjha28** [angelbroking-github/smartapi-python#11]
1. A user does not remember the Symbol token of every ticker instead he/she knows the SYMBOL. Please make sure Symbol Token should not be the primary key.

2. I am trying to create a system where user can specify every detail of the trade but symbol token is something not everybody remembers also there's no method to retrieve the Symbol token in the API.

**Comment by rjamitsharma01** [comment]
YOU are right @manjha28  but I think after making this library they forgot about it they never reply even 

every where they require symbol token which has no chance to get from anywhere apart from manualy getting it from broker terminal url


that is bull shit library I have ever seen
